### DISCUSSION TITLE
Can TB Profiler be used to profile sequencing data from NTM samples?

### DISCUSSION THREAD

**Discussion by harrismia** [jodyphelan/TBProfiler#172] [Ideas]
Hi Jody, 

 We have data from NTM samples that we would like to profiler for drug resistance mutations and get information about lineage etc.  Could TB Profiler be used for this ?   I assume we would need to specify the reference genome and the SNP database.  

     Thanks!

          Michael

**Comment by jodyphelan** [comment]
Yes TB-Profiler can be used for this. You would just have to specify the reference/gff and databases as in https://github.com/jodyphelan/tbdb/. I am currently workin on a tool which will support calling of species https://github.com/jodyphelan/NTM-Profiler. This is still in the early phase of development but eventually preformatted libraries should feature there for the common NTMs.

**Reply by harrismia** [reply]
Thanks very much for the information!   I will definitely check out NTM-Profiler and look into using TB Profiler for NTM with a custom tbdb and reference/gff.
